build.sbt

```diff
@@ -384,7 +383,7 @@ lazy val releaseSettings = Seq(
     inquireVersions,
     runClean,
     runTest,
-    releaseStepCommand(s"++${Scala211}"),
+    releaseStepCommand(s"++${Scala211}!"),
```

Comment:
What is this exclamation mark good for?

Reply:
This appears to restore the behaviour of `++` post 1.0.0 to the pre-1.0.0 behavior. I found a mention of it in the 1.0.0 release notes, under "Improvements". The PR which added it, from @jroper, is here https://github.com/sbt/sbt/pull/2613, with some relevant comments by @eed3si9n here https://github.com/sbt/sbt/pull/2613#issuecomment-219019160. I can't find any actual documentation of the `++` changes in general or the `!` suffix in particular.

Anyhow, adding it seems like the right thing to do.